### PR TITLE
fix: Retraining model for PyFunc E2E test

### PR DESCRIPTION
**Pull request by tiopramayudi**
# Description

Currently PyFunc E2E test is using pre-built model artifact, this will running fine when the dependency used for the model artifact is the same with the server. In scenario where dependency is different between artifact and the server, the artifact load will be failed in the PyFunc server, hence we need to retraining the model and using the output of that as artifact for PyFunc server in E2E script

# Modifications

Retraining xgboost and sklearn model 

# Tests

# Checklist
- [x] Added PR label
- [ ] Added unit test, integration, and/or e2e tests
- [ ] Tested locally
- [ ] Updated documentation
- [ ] Update Swagger spec if the PR introduce API changes
- [ ] Regenerated Golang and Python client if the PR introduces API changes

# Release Notes

```release-note

```

### CODE REVIEW

**Review by ariefrahmansyah — APPROVED** [review]
lgtm. thanks, tio!
